translations/README.md

```diff
@@ -2,10 +2,34 @@
 
 ## Content
 
+- [Supported Languages](#supported-languages)
+- [Adding new languages](#adding-new-languages)
 - [Export and import workflow](#export-and-import-workflow)
 - [Conversion between JSON, CSV and ODS](#conversion-between-json-csv-and-ods)
 - [Used file formats](#used-file-formats)
 
+## Supported Languages
+
+All supported languages and language tags can be viewed [here](src/config.ts) or in the [wiki](https://wiki.tuerantuer.org/integreat-languages).
+
+## Adding New Languages
+
+You need to follow several steps to add new languages:
+
+- Pick the correct tag for your language: https://iso639-3.sil.org/code_tables/639/data
+  - Normally the two letter [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1) language tag should be used.
```

Comment:
```suggestion
  - Usually the two letter [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1) language tag should be used.
  - Check websites in the target language like Wikipedia or government websites. This could reveal what people are used to in the language.
```